dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature

```diff
@@ -8,7 +8,7 @@ Subsystem-Name: JSON Web Token 1.0
 -features=com.ibm.wsspi.appserver.webBundleSecurity-1.0, \
   com.ibm.wsspi.appserver.webBundle-1.0, \
   com.ibm.websphere.appserver.servlet-3.0; ibm.tolerates:="3.1,4.0,5.0,6.0", \
-  io.openliberty.jwt1.0.internal.ee-6.0; ibm.tolerates:="9.0", \
+  io.openliberty.jwt1.0.internal.ee-6.0; ibm.tolerates:="10.0", \
```

Comment:
```suggestion
  io.openliberty.jwt1.0.internal.ee-6.0; ibm.tolerates:="9.0", \
```
If you don't do this or use "9.0,10.0", EE9 tests should be broken, but like I said in another comment the 10.0 one isn't needed.